RELEASE/scripts/CONSUME.ash

```diff
@@ -853,6 +859,14 @@ Diet get_diet(OrganSpace space, OrganSpace max, boolean nightcap)
 			if(space.fullness > 0)
 				break;
 		}
+		//Fill spleen again if stomache cleared spleen
+		//This ensures that if the booze also clears spleen, that wouldn't be wasted
+		if(space.spleen > 0) 
+		{
+			fill_spleen(d, space, max);
+			if(space.spleen > 0)
+			 break;
+		}
```

Comment:
Is this really necessary? Hitting an organ cleanser that would overcleanse an organ should automatically be triggering for that organ to be filled in first, if I recall correctly.

Reply:
I suspect it is necessary because of how we are manipulating `space.spleen` by temporarily buffering it to a larger value/size. This may not trigger the spleen to be "overcleansed".

The reason for the buffer is to actually allow spleen consumables to be used. If `spleen.space` was set to 0, we would never use any spleen consumables, even if we could clear said spleen usage with cleansers after

Reply:
But it has been years since I've looked at this, and I'm not sure how relevant this is anymore in view of the hobopolis consumable nerfs